### PR TITLE
ENYO-6120: Req: hides border-bottom of the Header

### PR DESCRIPTION
**Pull request by Sangwook-Lee**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [ ] Documentation was added or is not needed

* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
There're some cases that cannot use Header because of under line of that component.
Actually, GUI of some applications want to use Header without underline.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Added prop `hideLine` to hide `border-bottom` in the Header component.

### Links
[//]: # (Related issues, references)
ENYO-6120

### CODE REVIEW

File: packages/moonstone/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 The following is a curated list of changes in the Enact moonstone module, newest changes on the top.
 
+## unreleased
+
+### Added
+
+- `moonstone/Header` prop `hideLine` to hide `border-bottom`
```
Comment (webOS101) [inline packages/moonstone/CHANGELOG.md]: This is a bit too implementation-specific.  I'll fix it.

Comment (webOS101) [reply packages/moonstone/CHANGELOG.md]: Also, there is no `moonstone/Header` component.

Comment (Djspaceg) [reply packages/moonstone/CHANGELOG.md]: Nice change (collection of changes), @webOS101!

**Comment by webOS101** [comment]
Oops, forgot root CHANGELOG.  Fixed.

**Pull request merged**